apps/docs/PROMPT_GUIDE.md

```
@@ -0,0 +1,194 @@
+
+- **GitHub Copilot**: Type `#<filename>` to reference the prompt file.
+- **Cursor**: Use `@Files` and select your prompt file.
+- **Zed**: Use `/file` followed by your prompt's path.
+- **Windsurf**: Use `@Files` and choose your prompt file from the list.
+
+## Prompt
+
+````md
+(your full prompt content here — see "Writing the prompt" below)
+````
+````
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)